Package.swift

```diff
@@ -0,0 +1,25 @@
+// swift-tools-version: 5.7
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "NohanaImagePicker",
+    defaultLocalization: "en",
+    platforms: [
+        .iOS(.v13),
+        .macOS(.v11)
```

Comment:
I followed its previous supporting os. I'm not sure its versions are correct.
<img width="756" alt="스크린샷 2022-12-22 오전 9 42 54" src="https://user-images.githubusercontent.com/5136313/209031090-65424060-c63f-4d4f-b044-68ed64c012bc.png">